interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a set of commands that allows you to store and save your codes locally and online. Github is an online playform that stores your code, your codes history of edits, and collaboration between teams.
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you keep track of your codes history. Github is a cloud based platform that stores your code online in repositories. This not only saves your data but allows it to be accessed by others for collaboration
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: A boolean will return either a true or false statement. This allows for yes or no answers to be built in your code. A logical else if statement is a good example of this.
```

Comment:
This is correct, but specifically, things like relational, logical, and equality operators will return Boolean values.